FlorenceBML/BMLacq810/BMLacq810.xml

```diff
@@ -63,7 +63,7 @@
                 <explicit xml:lang="gez">ወአንብሮ፡ ውስተ፡ ቅድስት፡ ምስዋዒከ፡ መንፈሳዊት፡ ንጽሕት፡ መድኃት፡ ዘእምኃቤከ፡ እግዚእነ፡ ወመድኃኒነ፡ ኢየሱስ፡ ክርስ፡ ክርስቶስ።
                 ዘለከ፡ ስብሐት፡ ለዓለመ፡ ዓለም፡ አሜን።</explicit>
                 <note>According to <bibl><ptr target="bm:Marrassini1986Laurenziana"/><citedRange unit="page">112</citedRange></bibl>
-                  similar to <ref target="UppEt18">A.3</ref>.</note>
```

Comment:
I know this is not your work, but in the way I understand the guidelines, the references to all manuscript IDs outside of the record should be in the format 
`<ref type="mss" corresp="UppEt4">`
But I see now that in the old PR of this manuscript, the format `<ref target="BAVet74#ms_i1.3.2">` was suggested by @eu-genia https://github.com/BetaMasaheft/Manuscripts/pull/2049, and I am a bit confused :)

Reply:
I've gone through the record and changed systematically how file-external manuscripts are referred to